aptos-move/e2e-move-tests/src/harness.rs

```diff
@@ -259,6 +273,10 @@ impl MoveHarness {
         *seq_no_ref = seq_no + 1;
         account
             .transaction()
+            .chain_id(self.executor.get_chain_id())
+            .ttl(
+                self.executor.get_block_time() + 3_600_000_000, /* an hour after the current time */
+            )
```

Comment:
Might wanna double check if this is a good default, but so far seems like no tests are broken.